### PR TITLE
Feat/wellcome collection link (WIP) #5646 

### PR DESCRIPTION
**Pull request by creido-welly**

### CODE REVIEW

File: src/components/WellcomeCollectionBanner/_wellcome-collection-link.scss [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+}
+
+.wc-banner__link {
+  color: $wc-text;
+  font-family: var(--font-secondary);
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.wc-banner__link.wc-banner__link:active,
```
Comment (chriddell) [inline src/components/WellcomeCollectionBanner/_wellcome-collection-link.scss:39]: Why the double selector (`.wc-banner__link.wc-banner__link`)?

File: src/components/WellcomeCollectionBanner/WellcomeCollectionBanner.tsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { CookiesProvider, useCookies } from 'react-cookie';
+
+import Button from 'Button/Button';
+import Icon from 'Icon/Icon';
+
+export const WellcomeCollectionLink = () => {
+  const IS_WELLCOME_COLLECTION_LINK_SEEN = 'isWellcomeCollectionLinkSeen';
```
Comment (chriddell) [inline src/components/WellcomeCollectionBanner/WellcomeCollectionBanner.tsx:8]: Presuming the all caps are convention for cookie names/vars?

Comment (creido-welly) [reply src/components/WellcomeCollectionBanner/WellcomeCollectionBanner.tsx:8]: Yeah, all caps is a convention for declaring constants (not necessarily the same as `const`)

https://www.freecodecamp.org/news/when-to-capitalize-your-javascript-constants-4fabc0a4a4c4/

File: src/components/WellcomeCollectionBanner/_wellcome-collection-link.scss (hunk 1 above)
Comment (creido-welly) [reply src/components/WellcomeCollectionBanner/_wellcome-collection-link.scss:39]: Specificity added to override D7 styles

**Review by annikaC — APPROVED** [review]

**Review by chriddell — APPROVED** [review]

**Pull request merged**